From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 8 Oct 2014 09:42:33 +0200
Subject: Revert "s390/locking: Reenable optimistic spinning"
Patch-mainline: never, kabi

This reverts commit f008331068945523ff12ff13f6d07d7303e0e8a9, upstream
commit 36e7fdaa1a04fcf65b864232e1af56a51c7814d6. It adds a member to
struct mutex, so this breaks kABI and we have to revert it for now.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/s390/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/s390/Kconfig b/arch/s390/Kconfig
index faa97bd4948e..6671e8db1861 100644
--- a/arch/s390/Kconfig
+++ b/arch/s390/Kconfig
@@ -93,7 +93,6 @@ config S390
 	select ARCH_INLINE_WRITE_UNLOCK_IRQ
 	select ARCH_INLINE_WRITE_UNLOCK_IRQRESTORE
 	select ARCH_SAVE_PAGE_KEYS if HIBERNATION
-	select ARCH_SUPPORTS_ATOMIC_RMW
 	select ARCH_USE_CMPXCHG_LOCKREF
 	select ARCH_WANT_IPC_PARSE_VERSION
 	select BUILDTIME_EXTABLE_SORT
-- 
2.1.1

